src/Result/Invoice.php

```diff
@@ -61,16 +72,38 @@ public function getExpirationTime(): int
         return $this->getData()['expirationTime'];
     }
 
-    public function getMonitoringTime(): int
+    public function getMonitoringExpiration(): int
     {
-        return $this->getData()['monitoringTime'];
+        return $this->getData()['monitoringExpiration'];
     }
 
     public function isArchived(): bool
     {
         return $this->getData()['archived'];
     }
 
+    public function getCheckoutOptions(): InvoiceCheckoutOptions
+    {
+        $options = new InvoiceCheckoutOptions();
+        $options->setSpeedPolicy($this->getData()['checkout']['speedPolicy']);
+        $options->setPaymentMethods($this->getData()['checkout']['paymentMethods']);
+        $options->setExpirationMinutes($this->getData()['checkout']['expirationMinutes']);
+        $options->setMonitoringMinutes($this->getData()['checkout']['monitoringMinutes']);
+        $options->setPaymentTolerance($this->getData()['checkout']['paymentTolerance']);
+        $options->setRedirectURL($this->getData()['checkout']['redirectURL']);
+        $options->setRedirectAutomatically($this->getData()['checkout']['redirectAutomatically']);
+        $options->setRequiresRefundEmail($this->getData()['checkout']['requiresRefundEmail']);
+        $options->setDefaultLanguage($this->getData()['checkout']['defaultLanguage']);
+
+        return $options;
+    }
+
+    public function isPaid(): bool
```

Comment:
We removed this function with 2.0.0 as it is not clear what it means, also partial paid invoices return true here which is a bit of useless information. Also it does not cover when an invoice is paid late etc.

Reply:
Will take it out, not sure how it got in.

Reply:
I see you often merge master into your branch. On one of the other PRs yesterday some commits there were from Sept in the history, but no changes as they were already merged to master. So maybe your master is outdated and you merge it and it gets mixed. Could also be the reason why isPaid() came back. I always pull master from btcpayserver repo and then do a `git rebase master` but ymmv

Reply:
Fixed.